types.go

```diff
@@ -22,6 +22,8 @@ type ReleaseContent struct {
 type ReleaseHistory struct {
 	// AppVersion is the app version of the Helm Chart that has been deployed.
 	AppVersion string
+	// Description is the incident log about past Helm chart deployment issue.
```

Comment:
Is incident log helm terminology? This isn't correct e.g. if the helm release has been upgraded the text will be `Upgrade complete`.

This should describe what information is provided by the tiller API. Not how we want to use it in the operator.

Reply:
```	// Description is the log message about past Helm chart deployment provided by tiller API.``` WDYT?